module-1/lab-data_cleaning/your-code/main.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
Step 8: the merge of users and post df isn't working. Do you know why? On the 'on' attribute of merge function you need to specify the column on which you want to run the merge. You have included 'Merged' which is not a column in any of the dataframes. I encourage you to finish the lab when you have time.